init.go

```diff
@@ -17,6 +17,17 @@ const (
 	// EnvLogLevel is key of the environment variable that can be used to set the log
 	// level on Init.
 	EnvLogLevel = "SRC_LOG_LEVEL"
+	// EnvLogSamplingInitial is key of the environment variable that can be used to set
+	// the number of entries with identical messages to always output per second.
+	//
+	// Defaults to 100 - set explicitly to 0 or -1 to disable.
+	EnvLogSamplingInitial = "SRC_LOG_SAMPLING_INITIAL"
```

Comment:
Totally tangential, reading the code earlier made think that while it was acceptable to have the env var defined in that package, but now this is an external module, it's less the case. 

That being said, for a logging module that we've extracted, it's quite convenient and there's no need to be dogmatic there.

Reply:
My point wasn't about it being `SRC` specific, but that usually the best way to configure libraries is to let the callers deal with where they are getting the config from on their own and to take those settings from a constructor instead. 

Maybe that's something that we could bring back into the `init` package that you mentioned earlier. In any case, my comment here is mostly about general principles, we're good with how it presently is :)

Reply:
>My point wasn't about it being SRC specific, but that usually the best way to configure libraries is to let the callers deal with where they are getting the config from on their own and to take those settings from a constructor instead.

+1